packages/apex-node/src/coverageReporters/index.ts

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
Last review question on this @peternhale, more of a semantic one - could this be included under src/reporters? Should it be? Or do you feel this is distinct enough to merit the separation? Just thinking for future maintenance, we may wonder why we have two different reporter directories.